src/main/java/com/mide/gangsaeng/bannedword/BannedWordService.java

```diff
@@ -3,9 +3,9 @@
 import java.util.List;
 
 public interface BannedWordService {
-    boolean added(String word);
+    boolean add(String word);
     boolean delete(String word);
     List<BannedWord> list();
 
-    boolean findBannedWords(String content);
+    void findBannedWords(String content);
```

Comment:
I think we have to clarify method name.
Your `findBannedWords` feels like it will return the list of bannedWords.

Let's fix to `throwIfContainsBannedWords` or `validate` or `validateBannedWords` something like this

Reply:
I agree. I choose `validateBannedWords`.